packages/moonstone/DatePicker/DatePicker.js

```diff
@@ -63,7 +66,27 @@ const dateTimeConfig = {
 			.match(/([mdy]+)/ig)
 			.map(s => s[0].toLowerCase());
 
-		return {formatter, order};
+		/**
```

Comment:
This should not be a jsdoc comment block.  Comments can stay, but remove the 2nd `*` at the start.

Reply:
done!